operation/fhir-operation-bulkdata/src/main/java/org/linuxforhealth/fhir/operation/bulkdata/config/impl/AbstractSystemConfigurationImpl.java

```diff
@@ -107,7 +107,7 @@ private static final int defaultCoreMaxParititions() {
     }
 
     @Override
-    public String getCoreBatchIdEncryptionKey() {
+    public String getCoreBatchIdEncodingKey() {
         return FHIRConfigHelper.getStringProperty("fhirServer/bulkdata/core/batchIdEncryptionKey", null);
```

Comment:
I sort of wanted to change the config property name here to use "encoding" instead of "encryption" but that would break all the existing configs.  Think its worth it?

Reply:
Yes. I think breaking the config is beneficial in this case because it will draw attention to the change.

Reply:
done